docs/proposals/DataTable/README.md

```diff
@@ -0,0 +1,165 @@
+# DataTable
+
+The `DataTable` Component would give the users options for pagination, sorting,
+and column visibility by leveraging some sort of API or service.
```

Comment:
Maybe I missed some discussion somewhere but I don't see any mention of filtering. Was that cut from the scope?

Reply:
Is filtering a follow up?

Reply:
From my understanding filtering would be accomplished by leveraging the API, meaning the DataTable would not handle filtering internally, at least not in this iteration. 

any thoughts @jsuazod ?

Reply:
That is correct Gui, filtering on the client side could easily be on future iterations, for now, all the filters would need to be on the server and pass into the component data prop.

Reply:
I'm cool with this being a follow up. I was just thinking where there are some other client side things built in to this proposal (sorting, pagination) this would be included. All good